### DISCUSSION TITLE
Formato DateField

### DISCUSSION THREAD

**Discussion by filipe-mateus** [EuricoDNJR/SG-VILLA-DOLCE#58] [Q&A] [answered]
Recebi o determinado erro quando fui utilizar a rota de criação de estoque de produtos: 
`date/time field value out of range`
Meu formato de request:
`{
    "idProduto": "168d9d25-6ce7-4904-8f42-cbdeee98163d",
    "quantidade": 5,
    "dataEntrada": "11/10/2023",
    "dataVencimento": "12/12/2023",
    "observacoes": "Mercadoria adquirida por meio de brinde"
}`
Alguém sabe como resolver?

**Comment by EuricoDNJR** [comment, marked as answer]
Esse erro ocorre porque o campo DateField recebe por padrão no formato YYYY-MM-DD e você esta tentando com o formato DD/MM/YYYY.

Sua request ficaria assim:
`{ 
"idProduto": "168d9d25-6ce7-4904-8f42-cbdeee98163d", 
"quantidade": 5, 
"dataEntrada": "2023/10/11", 
"dataVencimento": "2023/12/12", 
"observacoes": "Mercadoria adquirida por meio de brinde"
 }`

**Reply by filipe-mateus** [reply]
mt obg
